packages/react/src/components/NumberInput/NumberInput-story.js

```diff
@@ -28,6 +35,7 @@ const props = () => ({
   max: number('Maximum value (max)', 100),
   value: number('Value (value)', 50),
   step: number('Step of up/down arrow (step)', 10),
+  size: select('Field size (size)', sizes, '') || undefined,
```

Comment:
I think we should handle default size value in the component. IIRC this is related to why we currently get some storybook console errors for invalid props in other components with size variants

Reply:
Makes sense. I’ll updat👍

Reply:
I don't seem to be getting any console warnings since it is undefined. It looks like the other components have an invalid default of `''`. Wondering if it makes more sense to just remove the default value across the components, that way we won't get any warnings. 

Will throw up another PR normalizing them for everyone to take a look at.

Reply:
https://github.com/carbon-design-system/carbon/blob/81785b265e5cc4d1c6bcf479456dfb77fe2664d2/packages/react/src/components/NumberInput/NumberInput-story.js#L25

https://github.com/carbon-design-system/carbon/blob/81785b265e5cc4d1c6bcf479456dfb77fe2664d2/packages/react/src/components/NumberInput/NumberInput-story.js#L38

can we directly set `undefined` instead of dealing with empty string? applies to the other components too

Reply:
Yeah seems like we can just do 

```
const sizes = {
  'Extra large size (xl)': 'xl',
  'Regular size (lg)': undefined,
  'Small size (sm)': 'sm',
};
```
```
size: select('Field size (size)', sizes, undefined) || undefined
```

Seems like a cleaner solution than dealing with an empty string or adding `lg`

Reply:
Hmm actually that seems to be causing console warnings when I hit `Show Info`

Reply:
Yeah seems like it may have been introduced when the Storybook was updated? Seeing it on the master branch. 

I'm going to go ahead with your suggestion here to just use `undefined` and put up another PR shortly